release-common-lib/pom.xml

```diff
@@ -5,13 +5,49 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.reactome.release</groupId>
 	<artifactId>release-common-lib</artifactId>
-	<version>1.1.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+	<version>1.1.1</version>
+	<name>Release Common Library</name>
 	<description>Common components used throughout the Release process.</description>
+	<url>https://reactome.org</url>
+
+	<licenses>
+		<license>
+			<name>GNU AFFERO GENERAL PUBLIC LICENSE</name>
+			<url>https://www.gnu.org/licenses/agpl-3.0.txt</url>
+		</license>
+	</licenses>
+
+	<developers>
+		<developer>
+			<name>Solomon Shorser</name>
+			<email>solomon.shorser@oicr.on.ca</email>
+			<organization>Reactome</organization>
+			<organizationUrl>https://reactome.org</organizationUrl>
+		</developer>
+		<developer>
+			<name>Joel Weiser</name>
+			<email>joel.weiser@oicr.on.ca</email>
+			<organization>Reactome</organization>
+			<organizationUrl>https://reactome.org</organizationUrl>
+		</developer>
+	</developers>
+
+	<scm>
+		<connection>scm:git:git://github.com/reactome/data-release-pipeline.git</connection>
```

Comment:
`git` is supposed to appear twice in a row here?

Reply:
Yup, here is the example from https://central.sonatype.org/pages/requirements.html#scm-information:
```
<scm>
  <connection>scm:git:git://github.com/simpligility/ossrh-demo.git</connection>
  <developerConnection>scm:git:ssh://github.com:simpligility/ossrh-demo.git</developerConnection>
  <url>http://github.com/simpligility/ossrh-demo/tree/master</url>
</scm>
```